entity-framework/core/modeling/generated-properties.md

```diff
@@ -62,54 +55,84 @@ You can configure any property to have its value generated for inserted entities
+Although stored computed columns seem like a good solution for managing last-updated timestamps, databases usually don't allow specifying functions such as `GETDATE()` in a computed column. As an alternative, you can set up a database trigger to achieve the same effect:
 
-### Computed columns
+```sql
+CREATE TRIGGER [dbo].[Blogs_UPDATE] ON [dbo].[Blogs]
+    AFTER UPDATE
+AS
+BEGIN
+    SET NOCOUNT ON;
 
-On most relational databases, a column can be configured to have its value computed in the database, typically with an expression referring to other columns:
+    IF ((SELECT TRIGGER_NESTLEVEL()) > 1) RETURN;
 
-[!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/ComputedColumn.cs?name=DefaultComputedColumn)]
+    DECLARE @Id INT
 
-The above creates a *virtual* computed column, whose value is computed every time it is fetched from the database. You may also specify that a computed column be *stored* (sometimes called *persisted*), meaning that it is computed on every update of the row, and is stored on disk alongside regular columns:
+    SELECT @Id = INSERTED.BlogId
+    FROM INSERTED
+
+    UPDATE dbo.Blogs
+    SET LastUpdated = GETDATE()
+    WHERE BlogId = @Id
+END
+```
+
+For information on creating triggers, [see the documentation on using raw SQL in migrations](xref:core/managing-schemas/migrations/managing#adding-raw-sql).
 
-[!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/ComputedColumn.cs?name=StoredComputedColumn)]
+## Overriding value generation
+
+Although a property is configured for value generation, in many cases you may still explicitly specify a value for it. Whether this will actually work depends on the specific value generation mechanism that has been configured; while you may specify an explicit value instead of using a column's default value, the same cannot be done with computed columns.
+
+To override value generation with an explicit value, simply set the property to any value that is not the CLR default value for that property's type (`null` for `string`, `0` for `int`, `Guid.Empty` for `Guid`, etc.).
 
 > [!NOTE]
-> Support for creating stored computed columns was added in EF Core 5.0.
+> Trying to insert explicit values into SQL Server IDENTITY fails by default; [see these docs for a workaround](xref:core/providers/sql-server/value-generation#inserting-explicit-values-into-identity-columns).
+
+To provide an explicit value for properties that have been configured as value generated on add or update, you must also configure the property as follows:
+
+[!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/ValueGeneratedOnAddOrUpdateWithPropertySaveBehavior.cs?name=ValueGeneratedOnAddOrUpdateWithPropertySaveBehavior&highlight=5)]
```

Comment:
Note that the original section in "Setting Explicit Values for Generated Properties" said to use PropertySaveBehavior.Ignore which didn't work for me - PropertySaveBehavior.Save does. Also, no exception was thrown when specifying Ignore (which is what the text said).

One last thing, when the property `ValueGeneratedOnAddOrUpdate()`, then overriding with explicit values works when *updating* (with PropertySaveBehavior.Save), but doesn't seem to work when *adding*...

Reply:
Opened https://github.com/dotnet/efcore/issues/23912 to track the apparent adding bug.